src/Dropdown.svelte

```diff
@@ -32,9 +32,9 @@
 </style>
 
 <div class="dropdown">
-  {#each items as {key, value, badge}}
-    {#if key && value}
-      <div class="dropdown-item" class:selected={value === selected} on:click={() => onSelect(value)} title={value}>{@html key}
+  {#each items as {key, value, badge, title}}
+    {#if key && value && title}
```

Comment:
?

Reply:
Hmm yeah we could make `title` required as an accessibility feature, so we don't have to check it here.